src/pages/ApproveAction/hooks/useFeeCustomizer.tsx

```diff
@@ -85,6 +86,16 @@ export function useFeeCustomizer({
   const [gasFeeModifier, setGasFeeModifier] = useState<GasFeeModifier>(
     GasFeeModifier.SLOW,
   );
+
+  useEffect(() => {
+    if (
+      network?.chainId === ChainId.AVALANCHE_MAINNET_ID ||
+      network?.chainId === ChainId.AVALANCHE_DEVNET_ID
+    ) {
```

Comment:
I believe it should be `AVALANCHE_TESTNET_ID` instead. Or better use the helper:

```suggestion
    if (network && isAvalancheNetwork(network)) {
```